manifests/j/Japplis/PostOnScreen/Pro/1.0/Japplis.PostOnScreen.Pro.locale.en-US.yaml

```
@@ -0,0 +1,28 @@
+# Created using Apache Ant script from Japplis
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Japplis.PostOnScreen.Pro
+PackageVersion: 1.0
+PackageLocale: en-US
+Publisher: Japplis
+PublisherUrl: https://www.japplis.com
+PackageName: Post On Screen Pro
+PackageUrl: https://www.japplis.com/post-on-screen/pro
+License: Commercial
+LicenseUrl: https://www.japplis.com/post-on-screen/pro/license.txt
+Copyright: Copyright © 2021 - 2023 Japplis. All rights reserved.
+ShortDescription: Display text and images on top of your screen
+ReleaseNotesUrl: https://www.japplis.com/post-on-screen/pro/changes.txt
+Tags: 
+- Post On Screen Pro
+- font
+- image
+- photo
+- on top
```

```suggestion
- post-on-screen-pro
- font
- image
- photo
- on-top
```
Tags should be written in lowercase with hypens used instead of spaces